impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/UnderwaterCity.java

```diff
@@ -14,8 +14,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating;
+package org.jboss.cdi.tck.tests.full.context.passivating;
 
+import jakarta.enterprise.context.Dependent;
+
+@Dependent
```

Comment:
This it a bit puzzling; it seems this class is not used anywhere...
In fact, `UnderwaterCity`, `CityInterface` and `CityDecorator` could all be just removed? WDYT @Ladicek?

Reply:
Yea agree, these 3 things don't seem to be used in this package. They are used in the `.broken.decorator.field` package.